src/main/java/com/minecolonies/coremod/tileentities/TileEntityWareHouse.java

```diff
@@ -376,7 +377,8 @@ private boolean isToolInHut(final String tool, @NotNull final AbstractBuilding r
                     }
                     else
                     {
-                        hasItem = InventoryUtils.isToolInProvider(entity, tool, requestingBuilding.getBuildingLevel());
+                        //TODO check that 0 i correct
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)